pkg/requester/jobtransform/wasm_stroage_spec.go

```diff
@@ -0,0 +1,27 @@
+package jobtransform
```

Comment:
Typo in filename (stroage->storage)

Reply:
fixed in [3037a12](https://github.com/bacalhau-project/bacalhau/pull/2719/commits/3037a12bda26e32657e585cd3add2a6e9afa81c4)